applications/working_with_helm_charts/understanding-helm.adoc

```diff
@@ -25,6 +26,12 @@ Helm provides the ability to:
 * Create your own charts with {product-title} or Kubernetes resources.
 * Package and share your applications as charts.
 
+[NOTE]
+====
+In {product-title} 4.10, Helm is disabled for the Multicluster Console (Technology Preview).
```

Comment:
Can we comment out this note till 4.10 GA? It does not make sense to publish this before 4.10 GA.